docs/DSA-Problem-Solution/Contains_duplicate.md

```diff
@@ -0,0 +1,98 @@
+---
+id: Contains-Duplicate-leetcode-217
```

Comment:
We use the `id` value as a unique `url`.  So, we need write it in small letters.